kratos.gid/apps/FluidDEM/xml/XmlController.tcl

```diff
@@ -78,7 +80,7 @@ proc FluidDEM::xml::CustomTree { args } {
 
     # set result_node [$root selectNodes "[spdAux::getRoute FLResults]/container\[@n = 'GiDOptions'\]"]
 	# if { $result_node ne "" } {$result_node delete}
-    spdAux::SetValueOnTreeItem state disabled FLSolStrat
+    spdAux::SetValueOnTreeItem state enabled FLSolStrat
```

Comment:
If we were disabling this item maybe it's not necessary to enable it explicitly and this line could be removed.